games/tic_tac_toe.c

```diff
@@ -28,6 +29,12 @@ int checkwin();  // checks everytime when a player or computer places 'X' or 'O'
  */
 static char game_table[9];
 
+/**
+ * stored_pos array will store posistion of that were already allocated by the user
+ * calloc(9,sizeof(int)) means an array of 9 elements.
+ */
+int *stored_pos = NULL;
```

Comment:
```suggestion
static unsigned char *stored_pos = NULL;
```
* `static` to recognize the variable as local to this file. 
* `unsigned char` pointer to be same as line 30
Is this even required? isnt it serving the same purpose as `game_table`?

Reply:
I can't actually remember what I have done here, but I think the purpose of `stored_pos` was to make sure that the values that was inserted by the user was not overwritten.